src/Individual Article Page/ArticlePage.jsx

```diff
@@ -121,9 +138,16 @@ const ArticlePage = () => {
               <h4>{comment.author} said: </h4>
               <p>{comment.body}</p>
               <p>at: {comment.created_at.slice(0, 10)}</p>
+              {changeUser === comment.author && (
+                <button onClick={() => handleDeleteClick(comment.comment_id)}>
+                  Delete Comment
+                </button>
+              )}
+              {comment.comment_id === deleteCommentId && deleted === true ? (<p>Commet deleted Successfully!</p>) : (null)}
```

Comment:
You have got this message in here, but I don't seem to see it when I delete a comment 🤔